installer/templates/new/widgets/text/text.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import Widget from '../../assets/javascripts/widget';
+import {updatedAt} from '../../assets/javascripts/helpers';
 
 import './text.scss';
 
 Widget.mount(class Text extends Widget {
   render() {
     return (
-      <div className={this.state.classname || this.props.className}>
+      <div className={this.props.className + " " + this.state.specialStyle}>
```

Comment:
I was doing this change and I found it a little confusing that we were going to call it `className` as it is being concatenated to something called `className` and the only distinction is one is in `props` and the other is in `state`.

I propose we call it `customClassName`